publishSqlCoreProject.yml

```diff
@@ -0,0 +1,46 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+
+trigger:
```

Comment:
This should probably be a part of the existing build/release pipeline - https://github.com/microsoft/sqltoolsservice/blob/main/azure-pipelines/build-and-release.yml

Reply:
I will integrate it in the main pipeline after I am done finalizing all the steps. Converting this to draft PR for now.